configs/server/forseti_conf_server.yaml.in

```diff
@@ -123,6 +123,13 @@ inventory:
         # Defaults to 3600 if not set.
         api_timeout: 3600
 
+
+        # Path to the CAI dump files. This is used when you have access to the
+        # dump files directly and would like forseti to parse them into the
+        # gcp_inventory table to use scanner / explainer on top of the data.
+        # iam_dump_file_gcs_path: ""
```

Comment:
Consider using a single key that takes a list of paths, then you can specify one or more files to import. The import logic doesn't care, and duplicate assets will be skipped if they happen to exist in multiple files.

This would be a more flexible approach than making the test data have to exactly match the number of files exported.

Reply:
updated to cai_dump_file_gcs_paths of type list.